test/conformance/kuttl/mutate/policy/cornercases/foreach-remove-elements/policy.yaml

```diff
@@ -15,6 +15,7 @@ spec:
       mutate:
         foreach:
           - list: request.object.spec.volumes[]
+            order: Descending
```

Comment:
How did this work before🤔

Reply:
It didn't ;)

Reply:
It didn't work...completely. See #7051 for an example of what didn't work.